src/components/dialogs/affiliation-error.vue

```diff
@@ -1,19 +1,36 @@
 <template>
-  <v-dialog v-model="getAffiliationErrorModalVisible" max-width="60%" persistent>
-    <v-card class="notify-dialog">
-      <v-card-title class="flex-column">
-        <v-icon large color="error" class="my-4">mdi-alert</v-icon>
-        <span>Error Affiliating Name Request</span>
+  <v-dialog id="affiliation-error-dialog" :value="isShowModal" max-width="45rem" persistent>
+    <v-card>
+      <v-card-title class="d-flex justify-space-between">
+        <div v-if="isUnableToStartRegistration">Unable to start registration</div>
+        <div v-if="isAssociatedOtherAccount">Name Request associated with another account</div>
+        <div class="mt-3">
+          <v-btn icon large class="dialog-close" @click="hideModal()">
+            <v-icon>mdi-close</v-icon>
+          </v-btn>
+        </div>
       </v-card-title>
 
-      <v-card-text class="text-center">
-        The specified name request may already be affiliated. Go to
-        <a :href="businessRegistryUrl">your Business Registry</a>
-        to use the name request.
+      <v-card-text class="copy-normal pt-8">
+        <p v-if="isUnableToStartRegistration" class="mb-0">
+          We encountered an error starting your registration. Try again later, or go to
+          <a :href="businessRegistryUrl">My Business Registry</a> and manually add this Name
```

Comment:
Normally i agree the same window makes sense but in Name Request, would a new tab make sense? 
Does the user need to reference any content here that might help after navigating to the registries home?

Reply:
That's a really good question.

In this "hand-holding" flow (to create a business from a NR), it makes sense to stay in the same window, but what you've pointed out an "error handling" scenario... In this case I think it makes sense to open My Business Registry in a new tab.

Fixed!